### ISSUE TITLE
Doesn't build?

### ISSUE THREAD

**Issue by bo3b** [nektra/Deviare-InProc#7]
If I do a git clone of the repository, the project structure does not seem to match the code.

Looking just at NktHookLib to be built, I get:
````
1>..\Lib\libudis86\amalgamation.c(33): fatal error C1083: Cannot open include file: 'source\libudis86\decode.h': No such file or directory
````

amalgamation.c has includes of:
````
#include "stdint.h"
#include "source\libudis86\decode.h"
#include "source\libudis86\extern.h"
````
And the path ````source\libudis86\```` does not exist.

Here is how the hierarchy looks:

![image](https://cloud.githubusercontent.com/assets/6544511/14557791/cf34de02-02b3-11e6-98cd-07bbefc6a192.png)

Decode.h is under ````tables```` not ````source```` and in any case is under ````libudis86``` not above it.

![image](https://cloud.githubusercontent.com/assets/6544511/14557841/14a5f70a-02b4-11e6-8a25-ae090cde51cd.png)

Visual Studio 2013.

I'm sure I can get it building, I just figured you might want to know.

**Comment by bo3b** [comment]
Hmmm... I see.  Submodules.  Not my favorite.

Please add a note to your front page ReadMe, that makes it clear that you have to do *git clone --recursive* for this to work.

Your instructions in INSTALLATION & USAGE are incomplete.

**Comment by scnale** [comment]
Fixed in 11b39d7ead74ce75ba.

**Comment by bo3b** [comment]
Cool, thanks for making that more clear.
